Source/WebCore/platform/graphics/gstreamer/WebKitWebSourceGStreamer.cpp

```diff
@@ -925,11 +925,11 @@ static void webKitWebSrcUriHandlerInit(gpointer gIface, gpointer)
     iface->set_uri = webKitWebSrcSetUri;
 }
 
-void webKitWebSrcSetResourceLoader(WebKitWebSrc* src, RefPtr<WebCore::PlatformMediaResourceLoader>&& loader)
+void webKitWebSrcSetResourceLoader(WebKitWebSrc* src, RefPtr<WebCore::PlatformMediaResourceLoader> loader)
```

Comment:
Shouldn't this be:

```suggestion
void webKitWebSrcSetResourceLoader(WebKitWebSrc* src, WebCore::PlatformMediaResourceLoader& loader)
```

? Passing a RefPtr by value is probably not desired?

Reply:
Why not?

Reply:
You'll have an unnecessary ref and unref

Reply:
I see no = operator in RefPtr for references. Passing a `const RefPtr<>&` would work though, I think?

Reply:
Sorry, I was incorrect to suggest using a raw reference, since this is a RefPtr, not a Ref. Should use a raw pointer instead.

There are two cases:

 * You want to move ownership. Use `Ref<Foo>&&` or `RefPtr<Foo>&&`
 * You want the caller to retain ownership. Use `Foo&` or `Foo*` to indicate ownership does not move.

The latter case applies here (even though you do eventually take ownership down below).